UnitsNet/Scripts/Include-GenerateQuantitySourceCode.ps1

```diff
@@ -189,12 +199,29 @@ namespace UnitsNet
         /// </summary>
         public static $unitEnumName BaseUnit => $unitEnumName.$baseUnitSingularName;
 
+"@; 
+    if($baseDimensions)
+    {
+@"
+        private static readonly BaseDimensions _baseDimensions = new BaseDimensions($baseDimensionLength, $baseDimensionMass, $baseDimensionTime, $baseDimensionElectricCurrent, $baseDimensionTemperature, $baseDimensionAmountOfSubstance, $baseDimensionLuminousIntensity);
+
+        /// <summary>
+        ///     The <see cref="BaseDimensions" /> of this quantity.
+        /// </summary>
+        public static BaseDimensions BaseDimensions
```

Comment:
We could remove backing field here too, statically initialized getter property. Again, feel free to do in separate PR. No biggie.